internal/namespaces/instance/v1/custom_server.go

```diff
@@ -407,13 +420,58 @@ func serverGetBuilder(c *core.Command) *core.Command {
 			})
 		}
 
+		volumes := map[string]*customVolume{}
+		blockAPI := block.NewAPI(client)
+
+		for _, volume := range getServerResp.Server.Volumes {
+			customVol := &customVolume{
+				ID:   volume.ID,
+				Zone: volume.Zone.String(),
+				Boot: volume.Boot,
+			}
+
+			blockVol, _ := blockAPI.GetVolume(&block.GetVolumeRequest{
+				VolumeID: volume.ID,
+				Zone:     volume.Zone,
+			})
+			if blockVol != nil {
+				customVol.Name = blockVol.Name
+				customVol.Size = blockVol.Size
+				customVol.VolumeType = blockVol.Type
+				customVol.State = blockVol.Status.String()
+				customVol.CreationDate = blockVol.CreatedAt
+				customVol.ModificationDate = blockVol.UpdatedAt
+				if *blockVol.Specs.PerfIops == 5000 {
```

Comment:
I would add a test to ensure Specs is not nil and I would transform it in a switch case to handle more easily different kind of block volume iops